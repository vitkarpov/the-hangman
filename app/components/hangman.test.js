const Hangman = require("./hangman");

describe("Hangman", () => {
  it("renders correctly", () => {
    expect(new Hangman({ misses: ["a", "b", "c"] }).render()).toMatchSnapshot();
  });
});
