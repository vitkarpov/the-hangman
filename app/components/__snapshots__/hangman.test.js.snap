// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hangman renders correctly 1`] = `
"
      <div class=\\"hangman-sprite\\"
        style=\\"background-position: 0 -768px;\\"
      ></div>
    "
`;
